### ISSUE TITLE
Show warning or error for capital letters in port names

### ISSUE THREAD

**Issue by timkrentz** [RIAPS/riaps-pycom#165]
A .riaps file with a port name containing a capital, such as
```
component foo{
    timer syncDelay;
}
```

will generate the handler in all lowercase, like

```
# riaps:keep_syncdelay:begin
    def on_syncdelay(self):
        pass
# riaps:keep_syncdelay:end
```

without any notification that the generated handler was changed. A warning or error here would be very useful as the runtime exceptions are not very clear.

**Comment by MMetelko** [comment]
This has been fixed in PR #164 for issue #161.  The case will remain as created in the model file for python components.

**Comment by timkrentz** [comment]
Ah, excellent. Thanks!
